src/Cake.Common/Tools/NUnit/NUnitAliases.cs

```diff
@@ -9,7 +9,14 @@
 namespace Cake.Common.Tools.NUnit
 {
     /// <summary>
-    /// Contains functionality related to running NUnit unit tests.
+    /// <para>Contains functionality related to running <see href="https://github.com/nunit/nunit">NUnit</see> tests.</para>
+    /// <para>
+    /// In order to use the commands for this alias, include the following in your build.cake file to download and
+    /// install from NuGet.org, or specify the ToolPath within the <see cref="NUnitSettings" /> class:
+    /// <code>
+    /// #tool "nuget:?package=NUnit.ConsoleRunner"
```

Comment:
@devlead yes, I think you are right.  To avoid any ambiguity, we should specify the correct nuget version for that particular set of aliases.